lib/build/build.js

```diff
@@ -72,10 +72,12 @@ class Build {
       await this._runStep('create workspace', () => this._createWorkspace())
       await this._runStep('read .peon.yml', () => this._readPeonConfig())
 
-      let { env, peonConfig } = this
+      let { peonConfig } = this
 
       if (peonConfig.cache && peonConfig.cache.length) {
-        await this._runStep('restore cache', () => this._restoreCache())
+        await this._runStep('restore cache', (updateOutput) =>
+          this._restoreCache(updateOutput)
+        )
```

Comment:
As shown below steps must now call updateOutput to set some output visible in status pages. We no longer use their return value for that.